test/models/normalize_timdex_test.rb

```diff
@@ -36,4 +36,32 @@ def aspace_records
     assert_match 'The Paul Earls archives contains a large number of music manuscript',
                  aspace_records['results'][0].blurb
   end
+
+  # Some ASpace records have a publication date range rather than a creation date range. The following regression test
+  # ensures that the NormalizeTimdex model does not error for such records.
+  test 'records with no creation date do not error' do
+    VCR.use_cassette('aspace publication date',
+                     allow_playback_repeats: true) do
+      raw_query = SearchTimdex.new.search('SOVEREIGN INTIMACY: PRIVATE MEDIA AND THE TRACES OF COLONIAL VIOLENCE')
+      assert_not raw_query['data']['search']['records'].first['dates'].first['kind'] == 'creation'
+      assert_nothing_raised do
+        NormalizeTimdex.new.to_result(raw_query, 'SOVEREIGN INTIMACY: PRIVATE MEDIA AND THE TRACES OF COLONIAL VIOLENCE')
+      end
+    end
+  end
+
+  # This is another regression test. Now that we know that records with no creation date will not error, we also want
```

Comment:
I like this additional test. Do you think we might want a test to confirm we don't error but still normalize _a_ date when more than one date exists now that we've decided yolo choose one instead of displaying none?

Reply:
Good point. Hmmm...

Reply:
The first record when searching for `Timurid Architecture Research Archive` has two creation date ranges that should work for this.